docs/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@loomjs/docs",
+  "version": "1.0.0",
+  "description": "Documentation for Loom",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "docz:dev": "docz dev",
+    "docz:build": "docz build"
+  },
+  "keywords": [
+    "loomjs"
+  ],
+  "author": "Marc Bramaud",
```

Comment:
should we define a generic team name for this? Like Loom Devs or something similar

Reply:
How about `authors` which is an array of owners? That's how redux does it.
Otherwise we remove this line entirely, like react does.

Reply:
In other PRs we used `Loom.js Contributers <team@loomjs.dev>`